app/narwos.py

```diff
@@ -112,10 +114,12 @@ def index():
 
     validated_files = os.listdir(
         PATH_TO_VALID_FILES)
+        
+    validated_files_to_show = [file for file in validated_files if file != '.gitkeep']
```

Comment:
If you decide to include such logic, you have to remember not only exclude these files from GUI, but also block to process them on API, but of course idea is ok